Gemfile

```diff
@@ -5,4 +5,6 @@ group :development do
   gem 'guard'
   gem 'guard-rspec'
   gem 'pry'
+  gem 'sequel'
+  gem 'pg'
```

Comment:
Does the README need to be updated with additional steps so that we can setup our local environment to run this app?